src/CloudyCooling.hpp

```diff
@@ -81,7 +81,7 @@ AMREX_GPU_HOST_DEVICE AMREX_FORCE_INLINE auto cloudy_cooling_function(Real const
 {
 	// interpolate cooling rates from Cloudy tables
 	const Real rhoH = rho * cloudy_H_mass_fraction; // mass density of H species
-	const Real nH = rhoH / quokka::hydrogen_mass_cgs;
+	const Real nH = rhoH / (C::m_p + C::m_e);
```

Comment:
Can you define a variable `quokka::m_h` for this quantity ($m_p + m_e$)? I think that would make this more readable.

Reply:
Nooo lol
I will do that in a separate PR if you really want it. Because we should then do it everywhere.

Reply:
How many times is it used? I don't see how it's that big of a change...

Reply:
We have also used it in a couple other test problems, and I want to make sure we can keep track of where we use m_H versus amu. I also find that tests take a bit longer to pass because of using microphysics constants (some cross communication lag between microphysics and quokka?). But it is much cleaner to do any changes needed for it in a separate PR.

Reply:
Ok, that's fine.